source/features/release-download-count.css

```diff
@@ -4,6 +4,11 @@
 	min-width: 6em;
```

Comment:
Is this the minimum we could have? The columns seem quite wide for the file size

Reply:
Without RGH:

<img width="316" alt="Screenshot 2024-07-24 at 1 08 21 AM" src="https://github.com/user-attachments/assets/f76f7063-ab96-4eba-b0fc-ed4c90678534">

With RGH:

<img width="386" alt="Screenshot 2024-07-24 at 1 26 31 AM" src="https://github.com/user-attachments/assets/98457fae-31e3-44f6-9fad-8084712b7251">

Probably not much gain making it narrower

Reply:
The first one is dynamic, our width are fixed. The gain comes when the filename is long and the viewport is wide.

Since we're adding 2em to one column, it'd make sense to reduce the other by 2em, if it fits.

Reply:
> For example it already is broken on 24.7.11

Broken on rgh.not.enabled too 😃 well I guess it's not too hard to fix

<img width="558" alt="Screenshot 2024-07-24 at 1 57 44 AM" src="https://github.com/user-attachments/assets/bb06a689-e766-42a0-ae1f-dbf097f2321c">

Reply:
I think grid/subgrid is the only safe solution here. Would you like to explore it? 

I think `display: grid` and `display: contents` might already be enough. `display: subgrid` might be easier to set up.

Reply:
For the purpose of this bug fix, I think this is enough:

```suggestion
	min-width: 4em;
```

It can be improved further separately. Let's just test and merge